plot_pixel_values.py

```diff
@@ -95,7 +143,20 @@ def get_opt():
                     ax.autoscale_view()
                 else:
                     ax.plot(x, y)
-
+                ax.annotate("{}".format(y.name),
+                            xy=(0.5, 0.5), xycoords="axes fraction",
+                            ha='center', va='center',
+                            color='lightgrey')
+                if opt.fit_scaling:
+                    nonzero = y != 0
+                    fit = fit_pix_values(dat['TEMPCD'][nonzero],
```

Comment:
The simulated data is great for getting a nice range of temperatures to check this fitting, but what should be done to make it more robust when we just don't have data yet for a good fit?